kernel/src/actions/visitors.rs

```diff
@@ -268,6 +251,59 @@ impl DataVisitor for RemoveVisitor {
     }
 }
 
+/// Get a DV out of some engine data. The column holding `storage_type` is specified as `storage_type_col_index`. We assume the calling code has
+/// extracted `storage_type` already to verify the DV data is there.
+pub(crate) fn visit_deletion_vector_at<'a>(
+    row_index: usize,
+    storage_type_col_index: usize,
+    storage_type: String,
+    getters: &[&'a dyn GetData<'a>],
+) -> DeltaResult<DeletionVectorDescriptor> {
+    let mut idx = storage_type_col_index + 1;
+    let path_or_inline_dv: String = getters[idx].get(row_index, "deletionVector.pathOrInlineDv")?;
+    idx += 1;
+    let offset: Option<i32> = getters[idx].get_opt(row_index, "deletionVector.offset")?;
```

Comment:
Given that other visitors just do raw indexing with magic constants, why not do it here as well? 
Caller just needs to pass e.g. `&getters[8..]` instead of `getters` (and no longer needs to pass `storage_type_col_index`)?

Reply:
Could even incorporate the is-present check that currently gets replicated at call sites, e.g.
```rust
        let deletion_vector = visit_deletion_vector_at(row_index, storage_type, &getters[7..])
            .transpose()?;
```

Reply:
yep, that's cleaner, thanks